tests/syft/core/node/node_test.py

```diff
@@ -151,14 +155,16 @@ def test_send_message_from_network_client_to_vm() -> None:
     bob_vm = sy.VirtualMachine(name="Bob")
     bob_vm_client = bob_vm.get_client()
     bob_vm.root_verify_key = bob_vm_client.verify_key  # inject 📡🔑 as 📍🗝
-    print(f"> {bob_vm.pprint} {bob_vm.keys}")
-    print(f"> {bob_vm_client.pprint} {bob_vm_client.keys}")
+    if sy.VERBOSE:
```

Comment:
thank you for sticking with the sy.VERBOSE flag

Reply:
I think we should change this to a logging library and set up some logging levels for everything so we can continue to build out more meaningful logging in a non disruptive way.